core/service/src/lib.rs

```diff
@@ -140,10 +140,12 @@ impl<Components> Service<Components>
 		let executor = NativeExecutor::new();
 
 		let mut keystore = Keystore::open(config.keystore_path.as_str().into())?;
+
+		// This is meant to be for testing only
+		// TODO: remove this
```

Comment:
`TODO`s no longer allowed in codebase without a corresponding item in the GH issue tracker.